src/lib/Libwin/passwd.c

```diff
@@ -1550,8 +1599,11 @@ isLocalAdminMember(char *user)
 	mbstowcs(userw, user, PBS_MAXHOSTNAME+UNLEN+1);
 	mbstowcs(gnamew, gname, GNLEN);
 
-	if (NetLocalGroupGetMembers(NULL, gnamew, 2, (LPBYTE *)&members,
-		MAX_PREFERRED_LENGTH, &nread, &totentries, NULL) != NERR_Success) {
+	status = NetLocalGroupGetMembers(NULL, gnamew, 2, (LPBYTE *)&members,
+				MAX_PREFERRED_LENGTH, &nread, &totentries, NULL);
+	if (status != NERR_Success) {
+		log_eventf(PBSEVENT_ERROR, 0, LOG_DEBUG, __func__, "failed in NetLocalGroupGetMembers, with errno %d",
```

Comment:
You might want to replace LOG_DEBUG with LOG_ERR and use server eventclass for the second param, for all such changes through out this PR.